app/src/org/commcare/activities/LoginActivity.java

```diff
@@ -114,6 +118,14 @@ public class LoginActivity extends CommCareActivity<LoginActivity>
     private boolean appLaunchedFromConnect;
     private boolean connectLaunchPerformed;
 
+    private BiometricManager biometricManager;

```

Comment:
Similar concerns here to CommCareSetupActivity, about encapsulation an repeating code related to unlocking ConnectID